### PR TITLE
MR1 - Tabs tray refresh

### PR DESCRIPTION
**Pull request by electricRGB**
Roux's Changes 
This PR merges the iPhone & iPad tab tray refresh work.

- Changes TabTray architecture to MVVM between TabTrayViewController and TabTrayViewModel
- Unifies tab tray operations under one controller
- Changes labelling of toggling Chon tabs from debug menu
- Move toolbars to TabTrayViewController and make it responsible for the toolbar actions

Nish's Extra Changes
Shortcut Changes:
- Removed Sync shortcut from library shortcut in home screen
- Removed see more for library shortcut

Library
- Sync panel is no longer shown in Library view and is moved to Tabs Tray

Colour
- Added new colour Green Shamrock

### CODE REVIEW

**Comment by electricRGB** [comment]
Still requires fixing UI tests before bitrise will pass.

**Comment by electricRGB** [comment]
TODO: remember to switch `bitrise.yml` file back to have `iPhone 11` in simulator device before merging.

**Comment by electricRGB** [comment]
GUYS! All the checks passed! I am SOOO happy! :)

**Comment by nbhasin2** [comment]
Whats left:

- [ ] Hooking up sync now buton
- [ ] Fixing test that might break due to removing sync library view 🤞🏼

**Comment by nbhasin2** [comment]
@dnarcese Can I get another set of eyes on this as its a big review from myself, Roux and our contributor. 
Thanks 🙏🏼

**Review by nbhasin2 — APPROVED** [review]

File: Client/Frontend/Browser/Tabs/TabTrayViewController.swift
Comment (dnarcese) [inline Client/Frontend/Browser/Tabs/TabTrayViewController.swift:156]: nit blank line

File: Client/Frontend/Browser/Tabs/TabTrayViewController.swift
Comment (dnarcese) [inline Client/Frontend/Browser/Tabs/TabTrayViewController.swift:269]: nit extra blank line

File: Client/Frontend/Browser/Tabs/TabTrayViewController.swift
Comment (dnarcese) [inline Client/Frontend/Browser/Tabs/TabTrayViewController.swift:364]: nit extra blank line

File: Client/Frontend/photon-colors.swift
Comment (dnarcese) [inline Client/Frontend/photon-colors.swift:85]: is there a way we can just use the system colour? Or do we have to define it ourselves

Comment (electricRGB) [reply Client/Frontend/photon-colors.swift:85]: We can use the system colours in iOS13+ and have them dynamically adjust based on the system theme, but not below. I was upset by this. This is how we found out that we support iOS 11.4. hahahaa. But also sadface.

**Comment by isabelrios** [comment]
@electricRGB please let us know if you need help with merging this due to test failures...

**Pull request merged**
